willsCheatMenu/src/style.scss

```diff
@@ -111,14 +114,6 @@
 		font-family: "Arvo", sans-serif;
 		font-size: 35px;
 		font-weight: 900;
-		&::before {
-			content: "Will's";
```

Comment:
:O

Reply:
i added a thing into the html so that "Will's" is on the same line

Reply:
i added a thing into the html so that "Will's" is on the same line

Reply:
How do you run a bookmarklet with wills cheat menu because ive got an school computer and the inspect console is blocked